main.cc

```diff
@@ -1,15 +1,11 @@
-#include <casacore/casa/OS/File.h>
 #include <casacore/casa/OS/HostInfo.h>
-#include <casacore/casa/Inputs/Input.h>
```

Comment:
casacore::Input is used in main, not util

Reply:
Thanks for reminding! I put `#include <casacore/casa/Inputs/Input.h>` back to `main.cc`.